client-data/js/board.js

```diff
@@ -62,8 +62,12 @@ Tools.connect = function () {
 		self.socket = null;
 	}
 
+	var url = new URL(window.location);
+	var params = new URLSearchParams(url.search);
+	var token = params.get("token");
 
 	this.socket = io.connect('', {
+		"query": "token=" + token,
```

Comment:
won't this add token=undefined in the (default) case where no token is defined ?

Reply:
Fixed in latest push